backend/tests/src/lib/common/enums/assertion/assertion-constants.enum.ts

```diff
@@ -0,0 +1,5 @@
+enum AssertionConstants {
```

Comment:
```suggestion
enum AssertionConstant {
```
Enum names should be singular. Don't forget to change the file name

Reply:
Fixed